funnel/jobs/jobs.py

```diff
@@ -1,11 +1,11 @@
+from flask import current_app
 from ..models import (db, TicketClient)
 from ..extapi.explara import ExplaraAPI
 from ..extapi.boxoffice import Boxoffice
-from funnel import app
 
 
 def import_tickets(ticket_client_id):
-    with app.test_request_context():
+    with current_app.test_request_context():
```

Comment:
Change this back to `app`. If you need a `test_request_context`, then `current_app` will be `None`, so this construct will fail. Do this everywhere that `test_request_context` or `app_context` is used.